<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RamjetEngineWidget</class>
 <widget class="QWidget" name="RamjetEngineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Reference Area</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>ft</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="maxMachLineEdit">
        <property name="toolTip">
         <string>The maximum operating mach for this engine. Thrust is zero beyond the maximum mach.</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Max Altitude</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="maxAltitudeLineEdit">
        <property name="toolTip">
         <string>The maximum operating altitude for this engine. Thrust is zero beyond the maximum altitude.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="designAltitudeLineEdit">
        <property name="toolTip">
         <string>The altitude at which the thrust is equal to the design thrust</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Max Mach</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="referenceAreaLineEdit">
        <property name="toolTip">
         <string>Aerodynamic drag coefficient-area product for the engine when it is inoperative</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Design Mach</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Design Thrust</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="designMachLineEdit">
        <property name="toolTip">
         <string>The mach number at which the thrust is equal to the design thrust</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Design Altitude</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="designThrustLineEdit">
        <property name="toolTip">
         <string>The thrust at the design altitude and design mach</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>lbs</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ft</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ft&lt;sup&gt;2&lt;/sup&gt;</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Tsfc</string>
        </property>
       </widget>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>lbs/hr</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="tsfcLineEdit">
        <property name="toolTip">
         <string>Thrust Specific Fuel Consumption</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Mass</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="massLineEdit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>lbs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>massLineEdit</tabstop>
  <tabstop>referenceAreaLineEdit</tabstop>
  <tabstop>designAltitudeLineEdit</tabstop>
  <tabstop>designMachLineEdit</tabstop>
  <tabstop>designThrustLineEdit</tabstop>
  <tabstop>maxAltitudeLineEdit</tabstop>
  <tabstop>maxMachLineEdit</tabstop>
  <tabstop>tsfcLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
